engine/proofview.ml

```diff
@@ -302,7 +302,8 @@ let tclONCE = Proof.once
 
 exception MoreThanOneSuccess
 let _ = CErrors.register_handler begin function
-  | MoreThanOneSuccess -> CErrors.user_err Pp.(str "This tactic has more than one success.")
+  | MoreThanOneSuccess ->
+    Pp.str "This tactic has more than one success."
```

Comment:
You probably want to not print the string directly, but use whatever function is used to print `UserError`, otherwise the error message will change. (unless user errors are now printed without any decoration, of course!)

Reply:
The printer for user error does not add any decoration so this does produce identical output.

[Decorations are added in the UI layer, as an after-printer step]

Reply:
Fair enough then :slightly_smiling_face: